blogposts/2020/release-3.24.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Sourcegraph 3.24 release"
+publishDate: 2021-01-20T10:00-07:00
+description: "Sourcegraph 3.24 is released."
+tags: [blog, release]
+slug: "release/3.24"
+published: true
+changelogItems:
+  - description: Added panels to the Sourcegraph monitoring dashboards
+    url: https://docs.sourcegraph.com/admin/observability/metrics#grafana
+    category: Admin
+  - description: Suggested filters in the search results page can now be scrolled.
+    url: https://github.com/sourcegraph/sourcegraph/pull/17097
+    category: Search
+  - description: Dashboard links now included in monitoring alerts.
+    url: https://docs.sourcegraph.com/admin/observability/alerting
+    category: Admin
+  - description: "Added `serviceKind` field of the `ExternalServiceKind` type to `Repository.externalURLs` GraphQL API, `serviceType` field is deprecated and will be removed in the future releases."
+    url: https://github.com/sourcegraph/sourcegraph/issues/14979
+    category: API
+  - description: "The minimum Kubernetes version required to use the [Kubernetes deployment option](https://docs.sourcegraph.com/admin/install/kubernetes) is now v1.15 (released June 2019)."
```

Comment:
Change to active voice: 

Kubernetes version v1.15 (released June 2019) or higher is now required to use the Kubernetes deployment option.